tests/test_webrtc_util.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for WebRTC utility."""
+
+import pytest
+from google_nest_sdm.webrtc_util import DIRECTION_SENDONLY, DIRECTION_SENDRECV, WebRTCUtility
+
+
+@pytest.fixture
+def utility():
+    """
+    Fixture to create a WebRTCUtility instance for testing.
+    """
+    return WebRTCUtility()
+
+def test_get_media_direction(utility: WebRTCUtility):
+    """
+    Test getting the direction in the SDP.
+    """
+    sdp = (
+        "v=0\r\n"
+        "o=- 123456 654321 IN IP4 127.0.0.1\r\n"
+        "s=Test\r\n"
+        "c=IN IP4 127.0.0.1\r\n"
+        "t=0 0\r\n"
+        "m=audio 49170 RTP/AVP 0\r\n"
+        "a=rtpmap:0 PCMU/8000\r\n"
+        "a=sendrecv\r\n"  # Existing direction
+        "m=video 51372 RTP/AVP 96\r\n"
+        "a=rtpmap:96 H264/90000\r\n"
+        "a=sendrecv\r\n"
+    )
+    
+    direction = utility.get_media_direction(sdp, 'audio')
+
+    assert direction == DIRECTION_SENDRECV
+
+def test_update_direction_in_answer(utility):
```

Comment:
What do you think about testing this through `fix_mozilla_sdp_answer` so it's exercising the interactions between the two functions? and I don't think it will hurt testability.